make-release-artifacts.sh

```diff
@@ -0,0 +1,26 @@
+# Clean dist directory
+rm -rf dist
+mkdir -p dist
+
+# Get new tags from remote
+git fetch --tags
+
+# Get latest tag name
+latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
Will this always be the latest tag? Could tag be specified?

Reply:
To keep the script simple, it will only release the latest tag. In most cases, we only need to sign and release each tag once. If required in the future, we can include functionality to allow tag selection, but that's gold-plating at the moment in my opinion.

Reply:
Fair enough just checking :)